packaging/kibana.spec

```diff
@@ -22,16 +22,32 @@ cd %_builddir
 rm -rf %{name}
 mkdir %{name}
 cd %{name}
+
 #extract sources
 tar xzf %_sourcedir/%{name}-%{version}.tar.gz
 if [ $? -ne 0 ]; then
    exit $?
 fi
 
+#extract dlumbrer/kbn_network plugin, following the renaming and file modification steps from github repo
+unzip -q resources/plugins/kbn_network*.zip -d plugins/
+if [ $? -ne 0 ]; then
+   exit $?
```

Comment:
Can we add an `echo` about the plugin unzip failing?

Reply:
Yeah good call

Reply:
Oh, I also checked that the quiet flag (`unzip -q`) doesn't suppress errors, so we'd still see those too.

Reply:
This exit code is going to return 0, because the echo above it will be successful.

Reply:
Updated